pkg/asset/agent/image/ignition.go

```diff
@@ -178,6 +175,11 @@ func (a *Ignition) Generate(dependencies asset.Parents) error {
 		return err
 	}
 
+	rendezvousHostFile := ignition.FileFromString(rendezvousHostEnvPath,
```

Comment:
Shouldn't the rendezvousHostFile should only be written if a nodeZeroIP is known? In the scenario where the config ISO is used, the nodeZeroIP should be unknown and no rendezvousHostFile would be written.

Reply:
Eventually, but we're not there yet: https://issues.redhat.com/browse/AGENT-560
That's why this is defined inline rather than in a file in data/data/agent/files though.